{'code': ExecutionFailed, 'message': [{"exit_code":1,"error_message":"Execution failed with error: [stderr]/azureml-envs/azureml_ba9520bf386d662001eeb9523395794e/lib/python3.6/site-packages/sklearn/linear_model/logistic.py:433: FutureWarning: Default solver will be changed to 'lbfgs' in 0.22. Specify a solver to silence this warning.\n[stderr]  FutureWarning)\nCleaning up all outstanding Run operations, waiting 300.0 seconds\n1 items cleaning up...\nCleanup took 0.06963753700256348 seconds\n[stderr]Traceback (most recent call last):\n[stderr]  

File \"train.py\", line 53, in <module>\n[stderr]    main()\n[stderr]  

File \"train.py\", line 44, in main\n[stderr]    model = LogisticRegression(C=args.C, max_iter=args.max_iter).fit(x_train, y_train)\n[stderr] 


 File \"/azureml-envs/azureml_ba9520bf386d662001eeb9523395794e/lib/python3.6/site-packages/sklearn/linear_model/logistic.py\", line 1288, in fit\n[stderr]    accept_large_sparse=solver != 'liblinear')\n[stderr] 

 File \"/azureml-envs/azureml_ba9520bf386d662001eeb9523395794e/lib/python3.6/site-packages/sklearn/utils/validation.py\", line 756, in check_X_y\n[stderr]    estimator=estimator)\n[stderr] 

 File \"/azureml-envs/azureml_ba9520bf386d662001eeb9523395794e/lib/python3.6/site-packages/sklearn/utils/validation.py\", line 573, in check_array\n[stderr]    allow_nan=force_all_finite == 'allow-nan')\n[stderr]  

File \"/azureml-envs/azureml_ba9520bf386d662001eeb9523395794e/lib/python3.6/site-packages/sklearn/utils/validation.py\", line 56, in _assert_all_finite\n[stderr]   

 raise ValueError(msg_err.format(type_err, X.dtype))\n[stderr]ValueError: Input contains NaN, infinity or a value too large for dtype('float64').\n[stderr]\n","process_name":"/azureml-envs/azureml_ba9520bf386d662001eeb9523395794e/bin/python","error_file":"user_logs/std_log.txt"}], 'target': , 'category': UserError, 'error_details': [{'key': exit_codes, 'value': 1}, ], 'inner_error': null}